backend/config/urls.py

```diff
@@ -11,6 +11,7 @@
     jwt_token_refresh,
     path("admin/", admin.site.urls),
     path("api/v1/accounts/", include("apps.accounts.api.v1.routes")),
+    path("api/v1/", include("apps.vacancies.api.v1.routes")),
```

Comment:
Keep the urls style above. Make it "api/v1/vacancies/" and inside your apps.vacancies.routes leave the first url empty.